typescript/infra/scripts/print-gas-prices.ts

```diff
@@ -48,6 +56,7 @@ async function main() {
 async function getGasPrice(
   mpp: MultiProtocolProvider,
   chain: string,
+  gasPrice?: GasPriceConfig,
```

Comment:
nit: just to avoid conflicts with line 65, maybe `oldGasPrice` or `existingGasPrice`?